app/controllers/api/physical_servers_controller.rb

```diff
@@ -64,7 +75,7 @@ def change_resource_state(state, type, id)
           server = resource_search(id, type, klass)
           desc = "Requested server state #{state} for #{server_ident(server)}"
           api_log_info(desc)
-          task_id = queue_object_action(server, desc, :method_name => state, :role => :ems_operations)
+          task_id = queue_object_action(server, desc, :method_name => state, :role => :ems_operations, :args => data)
```

Comment:
does :args default to a hash or an array ?  I see hash in 68 but MiqTask.generic_action_with_callback wants an array.

Reply:
Yes. Should be an array. You're right